resources/lang/fr/layout.php

```diff
@@ -222,7 +224,7 @@
     'popup_login' => [
         'login' => [
             'email' => 'adresse e-mail',
-            'forgot' => "J'ai oublié mes détails de connexion",
+            'forgot' => "J'ai oublié mes identifiants de connexion",
```

Comment:
This might be too long to fit
![screen shot 2017-07-28 at 15 10 03](https://user-images.githubusercontent.com/1694544/28704802-a78df832-73a7-11e7-8c5c-3df87d4d3630.png)